frontend/tests/features/wheresMyGene/shareLink.test.ts

```diff
@@ -22,50 +34,42 @@ const dataSets = [
     id: "db59611b-42de-4035-93aa-1ed39f38b467",
     text: "49 years old male - Fresh PBMCs (2 days post-intubation)",
   },
-  {
-    id: "eeacb0c1-2217-4cf6-b8ce-1f0fedf1b569",
-    text: "49 years old male - Fresh PBMCs (3 days post-intubation)",
-  },
-  {
-    id: "881fe679-c6e0-45a3-9427-c4e81be6921f",
-    text: "66 years old female - Fresh PBMCs (2 days post-intubation)",
-  },
-  {
-    id: "ea786a06-5855-48b7-80d7-0313a21a2044",
-    text: "66 years old female - Fresh PBMCs (3 days post-intubation)",
-  },
-  {
-    id: "456e8b9b-f872-488b-871d-94534090a865",
-    text: "Single-cell atlas of peripheral immune response to SARS-CoV-2 infection",
-  },
```

Comment:
remove these datasets, since having too many of them creates unpredictability in app state. E.g., in staging, applying these datasets would still result in blood and lung tissues. But in dev, it'd only have blood and no lung tissue